.github/workflows/deploy-dev.yaml

```diff
@@ -0,0 +1,19 @@
+name: dev_deployment
+
+on: workflow_call
+
+jobs:
+  deploy_dev:
+    runs-on: ubuntu-latest
+
+    env:
+      DEPLOYMENT_KEY: ${{ secrets.DEPLOYMENT_KEY_DEV }}
+      HEROKU_APP_NAME: ${{ secrets.HEROKU_APP_NAME_DEV }}
+      SHA: ${{ github.sha }}
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: 'deploy dev to heroku'
+        run: |
+          echo "Deploying to Heroku. To see progress, go to: https://dashboard.heroku.com/apps/$HEROKU_APP_NAME/activity"
```

Comment:
when I tested this I couldn't actually see this output in the test shell

Reply:
![image](https://github.com/torchbox/rca-wagtail-2019/assets/2128707/14f4619a-9c5d-4668-a944-0270ec441053)